### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by SanjoyanM**

### CODE REVIEW

**Review by dariomws — APPROVED** [review]
I'm ok after you add the parameter to the syntax section as @get-itips suggested. Thanks!

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2447,6 +2447,31 @@ Accept pipeline input: False
...
+
+If you need to, you can turn off this feature that supports a default sensitivity label for SharePoint document libraries. This is a tenant-level setting that requires you to use the Set-SPOTenant cmdlet with the DisableDocumentLibraryDefaultLabeling parameter set to True.
+
+After you run this command, you won't see the option to configure Default sensitivity labels for a document library. You won't be able to select a default sensitivity label for new or existing libraries.
+
+For document libraries that were previously configured for a default sensitivity label:
+
+1. The label selection for the document library remains but is deactivated so new files won't have the selected sensitivity label applied.
+2. Sensitivity labels that were applied as a default label aren't removed. 
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2460]:
```suggestion
2. Sensitivity labels that were applied as default labels aren't removed. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SanjoyanM, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@get-itips @JohanFreelancer9

**Review by get-itips — CHANGES_REQUESTED** [review]
BlockDownloadFileTypePolicy was added to the syntax but it is not in the parameters section

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SanjoyanM

**Review by SanjoyanM — COMMENTED** [review]
Thanks.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@SanjoyanM 
Hi,
Please resolve the file conflicts, and then @ mention me to merge this PR.
Thank you!

**Review by SanjoyanM — COMMENTED** [review]
agreed

**Comment by SanjoyanM** [comment]
@tiburd Please help merge this.

**Comment by tiburd** [comment]
@SanjoyanM 
If you need assistance, I recommend posting your question, and a link to this PR, in the PubOps [Ask an Admin Teams channel](https://aka.ms/askanadmin).
Thanks!

**Comment by yogkumgit** [comment]
@SanjoyanM Gentle reminder.

**Comment by SanjoyanM** [comment]
> @SanjoyanM Gentle reminder.

I am not sure what is pending on me !!?

**Comment by yogkumgit** [comment]
@tiburd

**Comment by tiburd** [comment]
@SanjoyanM 
Please resolve the file conflicts.
If you need assistance, you can post a question in the [Ask an Admin Teams channel](https://aka.ms/askanadmi).
Thanks!

**Comment by yogkumgit** [comment]
@SanjoyanM Gentle reminder.

**Comment by yogkumgit** [comment]
@SanjoyanM Please take the action as advised else we may proceed to close this PR. Will appreciate your prompt action. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please help. @SanjoyanM this PR is long pending, kindly either resolve the conflict or close the PR and create a new one if you like. Will appreciate your action. Thanks

**Pull request closed without merging**